### DISCUSSION TITLE
Use backrest on laptop

### DISCUSSION THREAD

**Discussion by Wakolipo** [garethgeorge/backrest#368] [Q&A]
Hi to all,
I just landed to backrest, after watching a youtube video of an expert Linux user, who strongly recommended this restic wrapping software.
I was wondering if it is suitable to be used on a laptop, typically not always turned on. I mean, If I set a daily backup of a certain folder, how does it work since It could happen I do not ever use the laptop within a day? Or what if I just use the laptop in the evening.
I used to write a _systemd_ timer unit that trigger a _rclone_ backup once the laptop turned on.
Have you any suggestion for using resbackrest  on laptop?
Many thanks in advance.

**Comment by garethgeorge** [comment]
Backrest takes reasonable steps to be suitable for laptop use. Primarily, backrest will run a backup at the scheduled time or at the next nearest time when the laptop is awake e.g. if the computer is asleep with backrest running in the background during a time when a backup would run, the backup will next run when you open the laptop. 

The only big caveat to be aware of is frequently rebooted systems: Backrest always initializes its backup schedule relative to the startup time so it will skip backups if the computer was not powered on when they were scheduled to run (health operations eg check and prune are relative  to their last run time so this isn’t a problem there).

**Reply by Wakolipo** [reply]
Many thanks for your quick reply.
I wrote _restic_ instead of _backrest_ in my previous post; my bad. 
Unfortunately, my laptop is often powered off when not in use. In my case, It would be useful if the backrest could check its last run, as prune does, and takes a new snapshot if none has been taken within the last day (or something similar).

**Reply by garethgeorge** [reply]
That makes sense, I think this is a reasonable item for a feature request if you'd like to file one.
